### PR TITLE
Add models translation

### PR DESCRIPTION
**Pull request by kasya**
- setup localization and translation
 - translate models for exam, question and user apps

### CODE REVIEW

File: apps/exam/models.py [hunk 1]
```diff
@@ -34,24 +45,25 @@ class ExamAttempt(models.Model):
...
+  duration_minutes = models.IntegerField(verbose_name=_('duration'), default=120)
+  grade = models.IntegerField(verbose_name=_('grade'), default=0)
+  mode = models.CharField(verbose_name=_('mode'), max_length=30,
                           default=PRACTICE_MODE,
                           choices=EXAM_MODES)
-  status = models.CharField(max_length=25,
+  status = models.CharField(verbose_name=_('status'), max_length=25,
                             default=STATUS_IN_PROGRESS,
                             choices=STATUSES)
   user = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
```
Comment (arkid15r) [inline apps/exam/models.py:69]: Do you want to cover FK and M2M fields as well?

File: apps/question/models.py [hunk 2]
```diff
@@ -1,13 +1,19 @@
 from django.db import models
+from django.utils.translation import gettext_lazy as _
 
 
 class Answer(models.Model):
   """Answer model."""
 
   text = models.CharField(max_length=1000)
```
Comment (arkid15r) [inline apps/question/models.py]: No translation here?

File: apps/user/templates/user/profile.html [hunk 3]
```diff
@@ -1,39 +1,42 @@
...
+  <h1 class="title">{% blocktranslate with firstname=user.first_name %}Welcome, {{ firstname }}!{% endblocktranslate %}
+  </h1>
   <hr>
-  Name: {{user.first_name}} <br>
-  Lastname: {{user.last_name}}<br>
-  Joined on: {{user.date_joined}}<br>
+  {% blocktranslate with firstname=user.first_name %}Name: {{ firstname }}{% endblocktranslate %} <br>
+  {% blocktranslate with lastname=user.last_name %}Lastname: {{ lastname }}{% endblocktranslate %}<br>
+  {% blocktranslate with date=user.date_joined %}Joined on: {{date}}{% endblocktranslate %}<br>
+  <br>
```
Comment (arkid15r) [inline apps/user/templates/user/profile.html]: Could those be just {% translate %} blocks here?

File: config/settings.py [hunk 4]
```diff
@@ -62,6 +64,7 @@
     'django.contrib.sessions.middleware.SessionMiddleware',
     'django.middleware.common.CommonMiddleware',
     'django.middleware.csrf.CsrfViewMiddleware',
+    #'django.middleware.locale.LocaleMiddleware',
```
Comment (arkid15r) [inline config/settings.py]: \#

File: config/settings.py [hunk 5]
```diff
@@ -159,14 +162,23 @@
...
 
 TIME_ZONE = 'UTC'
 
 USE_I18N = True
 
 USE_TZ = True
 
+DATETIME_FORMAT = 'N j, P'
+
+LOCALE_PATHS = [os.path.join(BASE_DIR, 'locale')]
```
Comment (arkid15r) [inline config/settings.py]:
```suggestion
LOCALE_PATHS = (os.path.join(BASE_DIR, 'locale'),)
```

File: locale/ru/LC_MESSAGES/django.po [hunk 6]
```diff
@@ -0,0 +1,300 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
```
Comment (arkid15r) [inline locale/ru/LC_MESSAGES/django.po]: I think this should be filled in too

File: locale/ru/LC_MESSAGES/django.po [hunk 7]
```diff
@@ -0,0 +1,300 @@
...
+#: apps/exam/models.py:31
+msgid "question count"
+msgstr "количество вопросов"
+
+#: apps/exam/models.py:35
+msgid "exam"
+msgstr "экзамен"
+
+#: apps/exam/models.py:36
+#, fuzzy
```
Comment (arkid15r) [inline locale/ru/LC_MESSAGES/django.po]: fuzzy

File: locale/ru/LC_MESSAGES/django.po [hunk 8]
```diff
@@ -0,0 +1,300 @@
...
+
+#: config/settings.py:168
+msgid "English"
+msgstr "Английский"
+
+#: config/settings.py:169
+msgid "Russian"
+msgstr "Русский"
+
+#~ msgid "finished"
```
Comment (arkid15r) [inline locale/ru/LC_MESSAGES/django.po]: Need to be cleaned up.

File: apps/exam/models.py (hunk 1 above)
Comment (kasya) [reply apps/exam/models.py:69]: @arkid15r  As I understood, translations for FK and M2M will be available from those models, as long as there are verbose names in Meta class. So there's no need to add it again. Is that not correct?

File: apps/question/models.py (hunk 2 above)
Comment (kasya) [reply apps/question/models.py]: Oops, missed it somehow.

File: apps/user/templates/user/profile.html (hunk 3 above)
Comment (kasya) [reply apps/user/templates/user/profile.html]: > Could those be just {% translate %} blocks here?

Not if there's a placeholder in a string.

Comment (kasya) [reply apps/user/templates/user/profile.html]: Switched to {% translate %}  like so:
`{% translate "Name:" %} {{ user.first_name }}`

**Review by arkid15r — APPROVED** [review]
LGTM

**Pull request merged**